ocaml/xapi/helpers.ml

```diff
@@ -1955,10 +1955,25 @@ end = struct
     Xapi_psr_util.load_psr_pool_secrets ()
 end
 
+let ( let@ ) f x = f x
+
+let with_temp_fd fd f = finally (fun () -> f fd) (fun () -> Unix.close fd)
+
+let with_temp_file ?mode prefix suffix f =
+  let temp_file, out_ch = Filename.open_temp_file ?mode prefix suffix in
```

Comment:
I would pick more symmetrical names: `tmp_path`, `tmp_channel` or similar. Because everything is in the local context of a temporary file, `temp` or `tmp` could be also dropped. Likewise below.

Reply:
done in the latest fixup